messageslog/info.json

```
@@ -3,8 +3,8 @@
     "Fixator10"
   ],
   "install_msg": "Thanks for install.",
-  "short": "Log deleted and redacted messages in defined channel",
-  "description": "Log deleted and redacted messages in defined channel",
+  "short": "Log deleted and edited messages in a defined channel",
+  "description": "Log deleted and edited messages in defined channel",
```

```suggestion
  "short": "Log deleted and edited messages in a defined channel",
  "description": "Log deleted and edited messages in a defined channel",
```

*English is not my native language, perhaps "to a defined channel" may be better here?*

---

Yep, I wasn't sure if you'd want additional language changes but that sounds right to me.